--- vamp-aubio-plugins-0.3.2/Makefile.orig	2007-06-17 22:12:08.000000000 +0000
+++ vamp-aubio-plugins-0.3.2/Makefile	2007-06-17 22:12:47.000000000 +0000
@@ -16,7 +16,7 @@
 # want to) statically link libstdc++, because our plugin exposes only
 # a C API so there are no boundary compatibility problems.
 #
-PLUGIN_LIBS	= -Wl,-Bstatic -lvamp-sdk -laubio -lfftw3f -Wl,-Bdynamic
+PLUGIN_LIBS	= -lvamp-sdk -laubio -lfftw3f -Wl,-Bdynamic
 
 # Flags required to tell the compiler to make a dynamically loadable object
 #
