--- vamp-aubio-plugins-0.3.2/plugins/Notes.cpp.orig	2007-06-17 22:00:19.000000000 +0000
+++ vamp-aubio-plugins-0.3.2/plugins/Notes.cpp	2007-06-17 22:03:17.000000000 +0000
@@ -395,7 +395,7 @@
     if (median < 45.0) return;
 
     float freq = median;
-    int midiPitch = (int)FLOOR(aubio_freqtomidi(freq) + 0.5);
+    int midiPitch = (int)floor(aubio_freqtomidi(freq) + 0.5);
     
     if (m_avoidLeaps) {
         if (m_prevPitch >= 0) {
